frontend/src/core/comments/components/AddComment.css

```diff
@@ -1,11 +1,36 @@
+.comments-list .add-comment form.container {
+    flex-flow: row nowrap;
+    align-items: flex-end;
+    padding-bottom: 5px;
+}
+
 .comments-list .add-comment textarea {
     background-color: #333941;
     border: none;
     border-radius: 4px;
     color: #FFFFFF;
+    display: flex;
+    align-items: center;
+    flex-grow: 2;
     font-size: 11px;
     height: auto;
     overflow: auto;
     line-height: 24px;
     padding: 6px;
 }
+
+.comments-list .add-comment .submit-button {
+    background-color: #4d5967;
+    border: solid 1px #4d5967;
+    border-radius: 4px;
+    color: #ccc;
+    display: flex;
+    align-items: center;
+    height: 36px;
+    margin-left: 6px;
+    padding: 0 12px;
+}
+
+.comments-list .add-comment .submit-button:hover {
+    color: #fff;
```

Comment:
Let's make the highlight a bit more obvious and in line with the other buttons of similar style on the dashboard:
```css
color: #272A2F;
background: #7BC876;
```

That's my last request for change. I'm working on the "Threaded view" of Translation comments, and once I'm done I plan to conduct a quick round of testing before shipping either of these two versions.